src/components/AccessibilityGuide/DynamicContent.stories.mdx

```diff
@@ -0,0 +1,275 @@
+import { Meta } from "@storybook/addon-docs";
+
+import { getCategory } from "../../utils/componentCategories";
+import DSProvider from "../../theme/provider";
+
+<Meta title={getCategory("Dynamic Content")} />
+
+# Dynamic Content
+
+## Table of Contents
+
+- [General Information](#general-information)
+- [ARIA Live Regions](#aria-live-regions)
+- [Managing Focus](#managing-focus)
+- [Anti-Patterns](#anti-patterns)
+- [NYPL Patterns](#nypl-patterns)
+- [Resources](#resources)
+
+## General Information
+
+Most websites have elements on that are updated dynamically. Typically, these
+updates are indicated with a visual loading animation before the new content is
+displayed. This visual cue, however, is not helpful for visually impaired users
+or users who use assistive technology. We need to ensure that the changes are
+understood and announced by the assistive technology.
+
+The quick takeaway for implementing this necessary functionality is to _always_
+render an empty container for the dynamic content in the DOM on the server-side.
+The container element should have an `aria-live="polite"` attribute applied. The
+following example shows one possible DOM structure that allows newly generated
+content to be recognized and announced by assistive technology.
+
+```html
+<div class="dynamic-content-container" aria-live="polite">
+  <!-- Any dynamic content should be inserted, removed, or updated here. -->
+</div>
+```
+
+## ARIA Live Regions
+
+The `aria-live` attribute is used to indicate that an element's content will be
+updated dynamically and that it should be announced to assistive technology.
+The `aria-live` attribute can be set to "off", "polite", or "assertive". _When_
+the content should be read by the assistive technology depends which of the three
```

Comment:
There are some grammatical issue in the end of this sentence.  It should be updated to the following:
```
_When_ the content should be read by the assistive technology 
depends on which of the three values is used.
```